explorer/site-config/ncpi-catalog/dev/index/studiesEntity.ts

```diff
@@ -18,6 +17,7 @@ export const studiesEntity: EntityConfig<NCPICatalogStudy> = {
     tabs: [],
     top: [],
   },
+  getId: getDbGapId,
   label: "Studies",
   list: {
     columns: [
```

Comment:
Check each sort key to ensure they match `NCPICatalogStudy` property names. Do the same check for all entities, for both AnVIL and NCPI.